pkgs/top-level/all-packages.nix

```diff
@@ -28289,6 +28289,8 @@ in
 
   xpdf = libsForQt5.callPackage ../applications/misc/xpdf { };
 
+  xplugd = callPackage ../tools/X11/xplugd {};
```

Comment:
```suggestion
  xplugd = callPackage ../tools/X11/xplugd { };
```

Reply:
By the way, have a read of the comment at the top:
```nix
/* The top-level package collection of nixpkgs.
 * It is sorted by categories corresponding to the folder names
 * in the /pkgs folder. Inside the categories packages are roughly
 * sorted by alphabet, but strict sorting has been long lost due
 * to merges. Please use the full-text search of your editor. ;)
 * Hint: ### starts category names.
 */
```

Reply:
Yeah, I’ve inserted before xpointerbarrier, which follows alphabetically in tools/X11. Thanks for the suggestion.

Reply:
Yeah, I meant that they are in clusters but they aren't in the correct categories (see the line with `### TOOLS`)